actors/miner/tests/util.rs

```diff
@@ -620,6 +620,16 @@ impl ActorHarness {
             })
             .collect();
 
+        for si in v2.iter() {
+            rt.expect_emitted_event(
+                EventBuilder::new()
+                    .typ("sector-precommitted")
+                    .field_indexed("miner", &RECEIVER_ID)
+                    .field_indexed("sector", &si.sector_number)
+                    .build()?,
```

Comment:
There's enough repeated code here that it's probably just worth adding a method to factor it out, like `expect_event(rt, "sector-precommitted", &RECEIVER_ID, &si.sector_number)`. This can be broadly useful because all the miner events have the same schema.

Reply:
Refactored this into it's function.